### ISSUE TITLE
Bug: App crashing when running as administrator and browsing for background image in Settings

### ISSUE THREAD

**Issue by What410802** [files-community/Files#15634]
### Description

Files crashes every time when running as administrator and click Settings -> Appearance -> Background image -> browse. The problem does not appear in normal user mode. 
debug.log: 
```
2024-06-20 22:53:54.5718|Information|App launched. Launch args type: LaunchActivatedEventArgs
2024-06-20 22:53:56.8702|Information|Drive added: C:\, Fixed
2024-06-20 22:53:57.2444|Information|Drive added: D:\, Fixed
2024-06-20 22:53:59.5592|Information|STORE: Checking for updates...
2024-06-20 23:02:21.8683|Error|System.Runtime.InteropServices.COMException (0x80004005)
   at Files.App.ViewModels.Settings.AppearanceViewModel.SelectBackgroundImage()
   at CommunityToolkit.Mvvm.Input.AsyncRelayCommand.AwaitAndThrowIfFailed(Task executionTask)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
   at Microsoft.UI.Dispatching.DispatcherQueueSynchronizationContext.<>c__DisplayClass2_0.<Post>b__0()
--- End of stack trace from previous location ---
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.Core.IUnhandledErrorMethods.Propagate(IObjectReference _obj)
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelperWinUI.<.ctor>b__0_3(Object sender, UnhandledErrorDetectedEventArgs eventArgs)
--- End of stack trace from previous location ---
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelperWinUI.<.ctor>b__0_3(Object sender, UnhandledErrorDetectedEventArgs eventArgs)
   at ABI.System.EventHandler`1.Do_Abi_Invoke[TAbi](Void* thisPtr, IntPtr sender, TAbi args)
```
![image](https://github.com/files-community/Files/assets/84611393/eac2593a-5e7a-4a8a-87bd-fc3dbcaa4434)

### Steps To Reproduce

1. Right click on a Files shortcut and choose Run as administrator
2. Click the Settings icon in the upper right corner
3. Choose Appearance
4. Click Browse in Background image

### Requirements

- Fix crash that occurs when browsing for background image in settings when running as administrator. 

### Files Version

3.4.1.0

### Windows Version

10.0.22623.1037

### Log File

[debug.log](https://github.com/user-attachments/files/15916340/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report, This is an issue with Microsoft WinAppSDK https://github.com/microsoft/WindowsAppSDK/issues/2504. There is no need to run Files as admin anyway.

**Comment by yaira2** [comment]
@What410802 thank you for reporting this issue, we're excited to share this is already resolved in the preview builds and will be included in the upcoming v3.5 release.
